src/topology/vector_bundle.lean

```diff
@@ -387,3 +411,57 @@ lemma is_open_map_proj : is_open_map Z.proj :=
 topological_fiber_bundle_core.is_open_map_proj Z
 
 end topological_vector_bundle_core
+
+end
+
+variable [∀ x, topological_space (E x)]
+
+open topological_space
+
+/-- This structure permits to define a vector bundle when trivializations are given as local
+equivalences but there is not yet a topology on the total space. The total space is hence given a
+topology in such a way that there is a fiber bundle structure for which the local equivalences
+are also local homeomorphism and hence vector bundle trivializations. -/
+@[nolint has_inhabited_instance]
+structure topological_vector_prebundle :=
+(trivialization_at : B → topological_vector_prebundle.trivialization R F E)
+(mem_base_trivialization_at : ∀ x : B, x ∈ (trivialization_at x).base_set)
+(continuous_triv_change : ∀ x y : B, continuous_on ((trivialization_at x) ∘
+  (trivialization_at y).to_local_equiv.symm) ((trivialization_at y).target ∩
+  ((trivialization_at y).to_local_equiv.symm ⁻¹' (trivialization_at x).source)))
+(inducing : ∀ (b : B), @inducing _ _ _ (⨆ x : B,
+  coinduced (trivialization_at x).to_prebundle_trivialization.set_symm (subtype.topological_space))
+  (λ x : (E b), (id ⟨b, x⟩ : total_space E)))
```

Comment:
an we do better here? Can we require something else which is easier to prove and from which we can prove this?

Reply:
Can't you just require that the composition of the inclusion and of the canonical chart is inducing? This should be much easier to check in the applications.